### DISCUSSION TITLE
Updating App to new version - keeps running

### DISCUSSION THREAD

**Discussion by SteveRobinson-FIShR** [cortexrd/Knack-Toolkit-Library#371] [Q&A]
Hi all

I have recently added KTL advanced features using the guide and carefully followed the instructions provided.  Upon install this seemed to be working fine yesterday but today when I have logged on I keep getting a message saying Updating App to new version and the screen refreshes?  Any idea what I have done wrong or how to disable this from happening?

Thanks in advance

Steve

**Comment by SteveRobinson-FIShR** [comment]
Hi All

I have managed to fix my issue which I believe was caused by setting the display app versions to false in the config and there being a conflict between the app version number and what was recorded on the system.  I am not sure what I have done was correct but it seems to have fixed the issue for now.  Once logged in I went to the sysop dashboard/sw update and changed the value to reflect what was appearing at the top RHS of screen and then hit broadcast.

If there is anything else I should have done I would be grateful for any assistance offered.

**Reply by amandajane-mo** [reply]
Hi Steve,

Welcome to the KTL.

We had a similar issue and it was down to a view being named incorrectly. 

https://github.com/cortexrd/Knack-Toolkit-Library/discussions/331

Might be worth going back through and checking your setup. If the issue still persists @cortexrd will be along to assist.

Amanda
